.github/workflows/hourly-clan-sync.yml

```
@@ -2,49 +2,60 @@ name: Hourly Clan Data Sync
 
 on:
   schedule:
-    # Run hourly
+    # Run every hour
     - cron: "0 * * * *"
 
 jobs:
   sync-data:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)